lib/chef/resource/yum_package.rb

```diff
@@ -62,6 +62,11 @@ class YumPackage < Chef::Resource::Package
       property :allow_downgrade, [ true, false ], default: false
 
       property :yum_binary, String
+
+      def get_package_name(x)
```

Comment:
Let's make this a private method. There really is no reason anyone else should call this method.   Also a small comment or yard doc about the intent of the function for future reference would be awesome.